treeherder/model/fields.py

```diff
@@ -34,3 +34,15 @@ def db_type(self, connection):
         if hasattr(rel_field, 'get_related_db_type'):
             return rel_field.get_related_db_type(connection)
         return super(FlexibleForeignKey, self).db_type(connection)
+
+
+# This is required to support BigAutoField
```

Comment:
Will Django 1.10's built-in bigautofield obviate the need for this sort of thing? If so, it might be nice to get that landed first.

https://bugzilla.mozilla.org/show_bug.cgi?id=1311967